code/web/src/modules/user/api/state.test.js

```diff
@@ -0,0 +1,112 @@
+import Item from '../../subscription/Item';
+import userReducer from './state';
+
+describe('Reducer Testing', () => {
+  let initState;
+  beforeEach(() => {
+    initState = {
+      error: null,
+      isLoading: false,
+      isAuthenticated: false,
+      details: null
+    }
+  })
+  it('should check if SET_USER', () => {
```

Comment:
The describe and it block setup is super similar to RSpec. I will probably be referencing this test some of the other syntactic details

Reply:
I'm really curious about back end testing.